### PR TITLE
Added support for OSS

### PR DESCRIPTION
**Pull request by jkiddo**

### CODE REVIEW

File: docker-compose/prepareHostEnv.sh [hunk 1]
```diff
@@ -138,9 +142,11 @@ copyFiles () {
     echo "Copying needed files to directories"
 
     echo "Artifactory configuration files"
-    if [ "$TYPE" == "pro" ]; then
+    if [ "$TYPE" == "pro" ] || [ "$TYPE" == "oss" ]; then
         cp -f ${SCRIPT_DIR}/../files/security/communication.key ${ROOT_DATA_DIR}/artifactory/etc
         cp -fr ${SCRIPT_DIR}/../files/access ${ROOT_DATA_DIR}/artifactory/
+#    elif [ "$TYPE" == "oss" ]; then
+#         cp -fr ${SCRIPT_DIR}/../files/access ${ROOT_DATA_DIR}/artifactory/
```
Comment (eldada) [inline docker-compose/prepareHostEnv.sh]: Please remove the commented lines

**Comment by jkiddo** [comment]
That is already included in the https://github.com/JFrogDev/artifactory-docker-examples/blob/master/docker-compose/README.md#artifactory-oss

**Comment by eldada** [comment]
You are right - the section exists, but an example of how to use `prepareHostEnv.sh` to setup the file system is not there. 
Similar to how we show it for **Artifactory Pro** (https://github.com/JFrogDev/artifactory-docker-examples/blob/master/docker-compose/README.md#artifactory-pro-with-postgresql-and-nginx-for-docker-registry-support)

**Comment by jkiddo** [comment]
updated

**Pull request merged**